src/rgw/rgw_zone_types.h

```diff
@@ -545,6 +545,7 @@ WRITE_CLASS_ENCODER(RGWZoneGroupPlacementTierS3)
 enum GlacierRestoreTierType : uint8_t {
   Standard = 0,
   Expedited = 1,
+  NoTier = 2,
```

Comment:
@cbodley @ivancich .. a new value is added to this enum which is part of `struct RGWZoneGroupTierS3Glacier` (below). We do not think this needs any version changes, but just checking with you to be sure. Please review.

[IMO upgrades won’t be affected. On downgrade, if a zone has this value set to NoTier, it’ll just map back to the default Standard.]